pkg/config/deployment.go

```diff
@@ -29,6 +29,10 @@ const (
 )
 
 type GenericDeploymentSpec struct {
+	Name   string            `json:"name"`
+	EnvID  string            `json:"envId"`
```

Comment:
Sorry, I was worried about is that an environment name isn't unique.

Reply:
It is possible to have multiple environments with the same name within a project, and I meant to say that in order to guarantee uniqueness of fields other than ID in Firestore, we need to have another document that stores only the environment name.
However, I think it is sufficient to leave that responsibility to the user for now.

Reply:
> However, I think it is sufficient to leave that responsibility to the user for now.

Yes. I think that is fine since the environment is going to be just a normal label eventually.
We will remove all environment requirements from Piped adding form, application adding/editing form.

Reply:
Totally!